src/main/java/be/appwise/core/networking/base/BaseRestClient.kt

```diff
@@ -1,203 +1,16 @@
 package be.appwise.core.networking.base
 
-import android.os.Build
-import android.provider.Settings
 import android.util.Log
-import be.appwise.core.core.CoreApp
-import be.appwise.core.networking.NetworkConstants
-import be.appwise.core.networking.Networking
-import be.appwise.core.networking.NetworkingUtil
-import be.appwise.core.networking.bagel.BagelInterceptor
-import be.appwise.core.networking.interceptors.Authenticator
-import be.appwise.core.networking.interceptors.HeaderInterceptor
-import be.appwise.core.networking.model.AccessToken
-import okhttp3.Interceptor
-import okhttp3.OkHttpClient
-import okhttp3.logging.HttpLoggingInterceptor
-import retrofit2.Retrofit
-import retrofit2.converter.gson.GsonConverterFactory
-import retrofit2.converter.scalars.ScalarsConverterFactory
 
-abstract class BaseRestClient<T> {
-    protected open val TAG = BaseRestClient::class.java.simpleName
+abstract class BaseRestClient<T>: BaseSimpleRestClient() {
```

Comment:
Is this class still needed? 🤔 In your screenshots you are using `BaseSimpleRestClient()`

Reply:
I kept in order to not have any breaking changes. If someone wants to use a RestClient with a fixed ApiService, they are still able to do so.

This could be removed completely, but that would mean I need some feedback from everyone.